lib/domain/model/extensions/search/search_extension.dart

```diff
@@ -12,4 +12,8 @@ extension SearchExtension on SearchModel {
       directChatMatrixID: directChatMatrixID
     );
   }
+    
+  bool searchDisplayName(String keyword) {
```

Comment:
you should choose another name for method, because it returns a bool type. IMO, change to `containsKeywordInDisplayName`

Reply:
done